src/decode.h

```diff
@@ -283,14 +283,14 @@ typedef struct rv_insn {
      * optimization enables the self-recursive function to reuse the same
      * function stack frame.
      *
-     * The @tailcall member indicates whether an intermediate representation
-     * (IR) is the final instruction in a basic block. The @impl member
-     * facilitates the direct invocation of the next instruction emulation
-     * without the need to compute the jump address. By utilizing these two
-     * members, all instruction emulations can be rewritten into a
+     * The @next member will indicate the next intermediate representation
```

Comment:
Why do you use "will indicate"?

Reply:
Sorry for the bad grammar. It is fixed now.